server/postgres/migrations/000002_add_xid_constraint.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE xids
+  DROP CONSTRAINT xids_owner_uid_key,
```

Comment:
This needs to be `DROP CONSTRAINT IF EXISTS` to function properly as a migration